catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/Field.js

```diff
@@ -111,7 +111,7 @@ define([
                     return;
                 }
             } else {
-                if (this.isEmpty() && !this.get('regex') && !this.get('required')) {
+                if (!this.get('value') && !this.get('regex') && !this.get('required')) {
```

Comment:
Ah, I didn't see that this was for an early return.  I'm a little interested in what the crux of the issue was, as that might clear things up.  Right now I'm guessing that lat / lon is of type Point, and based on the lines above this one, specifically this one: https://github.com/codice/ddf/blob/master/catalog/spatial/registry/registry-admin-modules/registry-admin-local-ui/src/main/webapp/js/model/Field.js#L103, I'm not sure how your change is changing anything (since those lines will set nonexistent point values to a blank array).